storage/migrations/mysql/4_campaign_logs.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE IF NOT EXISTS `campaign_failed_logs`
     `description` varchar(191)     NOT NULL,
     `created_at`  datetime(6),
     FOREIGN KEY (`user_id`) REFERENCES users (`id`)
-) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

Comment:
Why is this getting deleted? Check your IDE for any auto-format SQL things and remove those.

Reply:
i used to use the go fmt but he doesn't remove anything, i wont use any formats